src/cuda/cuda-conversion.cu

```diff
@@ -284,7 +286,7 @@ void rscuda::unpack_yuy2_cuda_helper(const uint8_t* h_src, uint8_t* h_dst, int n
 
     cudaDeviceSynchronize();
```

Comment:
In my opinion, this is unnecessary, but better to use async copy and use the same stream across the whole function and do a stream syncronization before we return from this function. By that we can avoid extra syncing, however the gain would be probably not very significant.  

https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__MEMORY.html#group__CUDART__MEMORY_1g85073372f776b4c4d5f89f7124b7bf79